### PR TITLE
Fixes sass loading

### PR DESCRIPTION
**Pull request by emiloberg**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var style = require('!style!css!sass!./style/globalStyle.scss');
+var style = require('./style/globalStyle.scss');
```
Comment (emiloberg) [inline src/index.js:1]: There are 2 ways of telling Webpack how to handle an imported file. The first way is to define loaders in your webpack config, e.g.

```
module.exports = {
    module: {
        loaders: [{
            test: /\.scss$/,
            loaders: 'ALL-LOADERS-HERE',
            exclude: /node_modules/
        }]
    }
};
```

The second way is to specify the loaders in the require statement itself. E.g. adding `style!css!sass` like you did;

```
require('!style!css!sass!./style/globalStyle.scss');
```

If you use this way (`style!css!sass`) of defining the loaders then those setting will override any loader settings you have in your webpack config. Therefor I do _strongly_ recommend that you _only_ use configuration in your webpack config file and do _not_ use the (`something!somethin-else`) require syntax!

File: webpack.config.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ const cssIdentifier = PRODUCTION ? '[hash:base64:10]' : '[path][name]---[local]'
 
 const cssLoader = PRODUCTION
 	?	ExtractTextPlugin.extract({
-			loader: 'css-loader?minimize&localIdentName=' + cssIdentifier
+			loader: 'css-loader?minimize&localIdentName=' + cssIdentifier + '!sass-loader?sourceMap'
```
Comment (emiloberg) [inline webpack.config.js:35]: You hadn't added the sass-loader to your production build. Here I added it.

File: webpack.config.js [hunk 3]
```diff
@@ -55,7 +55,7 @@ module.exports = {
     },
             {
                 test: /\.scss$/,
-                loaders: 'cssLoader',
+                loaders: cssLoader,
```
Comment (emiloberg) [inline webpack.config.js:58]: You're using the **string** `cssLoader` as the loader, which isn't correct. You want to use the **variable** `cssLoader`.. Notice no apostrophe.

Comment (bogdani) [reply webpack.config.js:58]: Thank you for pointing out my mistakes, i've modified the files, but now i'm getting this errors on build
[error.txt](https://github.com/bogdani/workflow/files/583284/error.txt)

**Review by bogdani — APPROVED** [review]

**Pull request merged**